### ISSUE TITLE
Geting loss of NaN for some epochs during training

### ISSUE THREAD

**Issue by golnazhabibi3** [abduallahmohamed/Social-STGCNN#20]
I have got loss of nan during my training. Update: I think it should be some error dataset I have used. so i close this issue..

**Comment by yeyingdege** [comment]
> I have got loss of nan during my training. Update: I think it should be some error in the dataset I have used. so I close this issue..

@golnazhabibi3 Hi, I got all the loss of nan after epoch 3. My dataset seems fine. Have you found the problem?

**Comment by daeunni** [comment]
> > I have got loss of nan during my training. Update: I think it should be some error in the dataset I have used. so I close this issue..
> 
> @golnazhabibi3 Hi, I got all the loss of nan after epoch 3. My dataset seems fine. Have you found the problem?

Hi, I have same problem with you. Are you found cause of your problem? plz let me know

**Comment by xzf87845804** [comment]
@golnazhabibi3 Hi, I have the same problem, have you solved it?

**Comment by xzf87845804** [comment]
@yeyingdege Hi, I have the same problem, have you solved it?

**Comment by CANDY-HOUSE2002** [comment]
@xzf87845804 I try to reduce the learning_rate, it works!
